client/src/components/CheckboxSetField/CheckboxSetField.js

```diff
@@ -129,13 +129,18 @@ CheckboxSetField.propTypes = {
   disabled: PropTypes.bool,
 };
 
-CheckboxSetField.defaultProps = {
+CheckboxSet.defaultProps = {
   // React considers "undefined" as an uncontrolled component.
   extraClass: '',
   className: '',
   value: [],
 };
 
-export { CheckboxSetField as Component };
+export { CheckboxSet as Component };
```

Comment:
Ok so because we're exporting this `as Component`, the fact that we're renaming the component doesn't constitute a major change, right?

I personally don't feel that the renaming of this and the CheckboxField are warranted, because the CheckboxWithHolder pattern is deliberately indicating that it has the FieldHolder HOC applied to it, but if this can be done without breaking semver then I'm ambivalent about it.